ios/Podfile

```
@@ -39,9 +39,10 @@ end
 
 target 'NotificationServiceExtension' do
   use_frameworks!
-  pod 'CustomerIO/MessagingPushFCM', '~> 2.1.0'
+  pod 'CustomerIO/MessagingPushFCM', '~> 2.1.1'
```

```suggestion
  pod 'CustomerIO/MessagingPushFCM', '~> 2.1'
```

This syntax will automatically upgrade the CIO SDK to the latest version 2 release. This is the preferred syntax we recommend to all customers.